src/test/java/com/phonebookservice/controller/TestContactController.java

```diff
@@ -0,0 +1,132 @@
+package test.java.com.phonebookservice.controller;
```

Comment:
`src/test` ?? You shouldn't add test folder inside the src folder.

Reply:
you are right but the unit test report default path is inside of src.
I am checking to how to update the path